tests/service/ParserTests.fs

```diff
@@ -101,4 +101,31 @@ match () with
     match getSingleExprInModule parseResults with
     | SynExpr.Match (_, _, [ SynMatchClause (_, _, _, SynExpr.ArbitraryAfterError _, _, _)
                              SynMatchClause (_, _, _, SynExpr.Const _, _, _) ], _) -> ()
-    | _ -> failwith "Unexpected tree"
\ No newline at end of file
+    | _ -> failwith "Unexpected tree"
+
+[<Test>]
+let ``Let - Parameter - Paren 01`` () =
+    let parseResults = getParseResults """
+let f (x
+"""
+
+    match getSingleDeclInModule parseResults with
+    | SynModuleDecl.Let (_, [ SynBinding (headPat = headPat) ], _) ->
+        match headPat with
+        | SynPat.LongIdent (_, _, _, SynArgPats.Pats [ SynPat.FromParseError (SynPat.Paren (SynPat.Named _, _), _) ], _, _) -> ()
+        | _ -> failwith "Unexpected tree"
+    | _ -> failwith "Unexpected tree"
+
+[<Test>]
+let ``Let - Parameter - Paren 02 - Tuple`` () =
+    let parseResults = getParseResults """
+let f (x, y
```

Comment:
Can you add a test for 

```
let f (x,
```

thanks

Reply:
Also maybe
```
match () with
| (x, 
| _ -> ()
```
and
```
match () with
| (x
| _ -> ()
```

Reply:
@dsyme Added for all cases. I can't say I like an extra pattern in the first case much, but it's probably OK here?